desktop/cmp/rest/data/RestStore.ts

```diff
@@ -4,44 +4,57 @@
  *
  * Copyright © 2022 Extremely Heavy Industries Inc.
  */
-import {XH} from '@xh/hoist/core';
-import {UrlStore} from '@xh/hoist/data';
+import {PlainObject, XH} from '@xh/hoist/core';
+import {
+    StoreRecord,
+    UrlStore,
+    UrlStoreConfig
+} from '@xh/hoist/data';
 import '@xh/hoist/desktop/register';
 import {filter, keyBy, mapValues} from 'lodash';
-import {RestField} from './RestField';
+import {RestField, RestFieldSpec} from './RestField';
+
+
+export interface RestStoreConfig extends UrlStoreConfig {
+
+    /** Field names, configs, or instances. */
+    fields?: (string|RestFieldSpec|RestField)[];
```

Comment:
Nice that we are able to type this override and pair it with defaultFieldClass. Should we also override fields on the RestStore class below to be typed as `fields: RestField[]`?

Reply:
Done!  Needed to use 'declare' keyword, but it worked great.